cassandra-mesos-scheduler/src/main/java/io/mesosphere/mesos/frameworks/cassandra/scheduler/CassandraScheduler.java

```diff
@@ -88,6 +93,9 @@ public void resourceOffers(final SchedulerDriver driver, final List<Offer> offer
         if (LOGGER.isDebugEnabled()) {
             LOGGER.debug("> resourceOffers(driver : {}, offers : {})", driver, protoToString(offers));
         }
+
+        configuration.getSeedManager().syncSeeds(clock.now().getMillis());
```

Comment:
IMHO, this is engineering overkill. It will not add anything useful, except additional complexity.
The overhead of evaluating the check "do we need to fetch" is minimal. Let me collect the actual numbers ...